tool/canonical/scorecard.dart

```diff
@@ -60,62 +59,6 @@ Iterable<LintRule>? get registeredLints {
 
 List<String?> get unfixableLints => _unfixableLints ?? _getUnfixableLints();
 
-StringBuffer buildFooter(ScoreCard scorecard, List<Detail> details) {
```

Comment:
This is reachable if uncommented in `main`.

Reply:
Apologies, fixed and changed commented code to be guarded with a condition.